locale/ManageIQ_Providers_Azure.pot

```diff
@@ -0,0 +1,155 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the ManageIQ_Providers_Azure package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```

Comment:
Hmm, don't know. Things will work just fine even without this info being filled in :smile: 

I'd say this is just a cosmetic issue, but I can definitely fix this if it's important.